app/frontend/shared/utils/DOMPurify.res

```diff
@@ -5,15 +5,33 @@ type dompurify
 
 let sanitize = s => sanitizeExternal(dompurify, s)
 
-type options = {"ALLOWED_TAGS": array<string>}
+type optionalStringArray = option<array<string>>
```

Comment:
Making an array optional feels a bit wired 🤔

Reply:
Check this out https://github.com/pupilfirst/pupilfirst/pull/1119/commits/4cbe41d07cb89750e7ed3f546aafb2fe0ffb82dc